src/lib/integration_test_local_engine/docker_compose.ml

```diff
@@ -0,0 +1,63 @@
+open Core
+
+module Compose = struct
+  module DockerMap = struct
+    type 'a t = (string, 'a, String.comparator_witness) Map.t
```

Comment:
This is better summarized as `type 'a t = 'a String.Map.t`.

Given that this module only has this type def and the `empty` value, you should probably just delete this module and use `String.Map` directly (it has both `'a t` and `empty` on it already).